commit e14b3d582e1d9d338b7082d61f34ba8d1bbc540a (refs/changes/14/714/4)
Author: Marsell Kukuljevic <marsell@joyent.com>
Date:   2016-10-18T12:39:48+13:00 (3 years ago)
    
    TOOLS-1509: Update contribution guidelines to reflect use of cr.joyent.us
    Reviewed by: Joshua M. Clulow <jmc@joyent.com>
    Approved by: Joshua M. Clulow <jmc@joyent.com>

diff --git a/README.md b/README.md
index 9f8c790..2103c38 100644
--- a/README.md
+++ b/README.md
@@ -10,15 +10,15 @@
 
 # keyapi
 
-This repository is part of the Joyent SmartDataCenter project (SDC).  For 
-contribution guidelines, issues, and general documentation, visit the main
-[SDC](http://github.com/joyent/sdc) project page.
-
+This repository is part of the Joyent Triton project. See the [contribution
+guidelines](https://github.com/joyent/triton/blob/master/CONTRIBUTING.md) --
+*Triton does not use GitHub PRs* -- and general documentation at the main
+[Triton project](https://github.com/joyent/triton) page.
 
 KeyAPI is a simple library to create and unpackage opaque encrypted tokens
 for services that require user authentication
 
-KeyAPI provides a mechanism for SDC to securely pass information in and out
+KeyAPI provides a mechanism for Triton to securely pass information in and out
 of the system using tokens.
 
 # Using KeyAPI
